bv-evaluation/collect-stats-bv-decide.py

```diff
@@ -416,22 +417,19 @@ def slowdown_smtlib_stats():
 
 
 def smtlib_stats(performance_smtlib_dir):
-    df_bitwuzla_46k = pd.read_csv("../raw-data/SMTLIB/bitwuzla_46k.csv")
-    df_leanwuzla_46k = pd.read_csv("../raw-data/SMTLIB/leanwuzla_46k.csv")
-    assert len(df_bitwuzla_46k) == len(df_leanwuzla_46k)
+    df_bitwuzla = pd.read_csv("../raw-data/SMT-LIB/bitwuzla_data.csv")
```

Comment:
```suggestion
    df_bitwuzla = pd.read_csv("../raw-data/SMT-LIB/bitwuzla_data.csv")
```
sorry just a double check - `raw-data` should now be at the same level of the `collect-stats-script`, is the `../` necessary?

Reply:
No. I removed `../` in my local workspace. I didn't know you already fixed this issue (~8 mins ago)!

Reply:
Resolved!

Reply:
thanks :)